src/qilib/configuration_helper/adapters/ami430_instrument_adapter.py

```diff
@@ -46,15 +46,23 @@ def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
     def apply(self, config: PythonJsonStructure) -> None:
         """ Does not apply config only  compares config to device settings.
 
+        Configuration Parameter with setter commands are matched for equality, on mismatch
+        error is raised
+
         Args:
             config: Containing the instrument configuration.
-
         """
-        parameter_argument= {'field': self.field_variation_tolerance}
-        super().compare_config_on_apply(config, parameter_argument)
+        super()._config_with_setter_command_mismatch_raises_error(config)
```

Comment:
1. super() not self() : The way to access parameter is same, so put it in super(), we can do with self() also but it would be duplicate code to access parameter
2. Made it _apply